server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/EventDeadLettersIntegrationTest.java

```diff
@@ -463,6 +464,7 @@ public void multipleFailedEventsShouldBeCorrectlyProcessedByListenerAfterSuccess
         awaitAtMostTenSeconds.until(() -> retryEventsListener.getSuccessfulEvents().size() == 2);
     }
 
+    @Ignore("retry rest API delivers only once, see JAMES-2907. We need same retry cound for this test to work")
```

Comment:
s/cound/count/

Reply:
Jira?

Reply:
the JIRA reference is here, what do you mean?

Reply:
I find it visually easier to add the JIRA as a prefix. That ok to me as is.